common/v2/features/DeFiZap/components/ZapForm.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import { Button, AccountDropdown, InlineMessage, AmountInput } from 'v2/components';
+import { ZapInteractionState, ISimpleTxFormFull } from '../types';
+import styled from 'styled-components';
+import { Formik, Form, Field, FieldProps } from 'formik';
+import translate from 'v2/translations';
+import { IAccount, ExtendedAsset, Network } from 'v2/types';
+import { StoreContext, AssetContext, getNonce, NetworkContext, fetchGasPriceEstimates } from 'v2';
+import { validateAmountField } from 'v2/features/SendAssets/components/validators/validators';
+import { isEthereumAccount } from 'v2/services/Store/Account/helpers';
+
+interface Props extends ZapInteractionState {
+  onComplete(fields: any): void;
+  handleUserInputFormSubmit(fields: any): void;
+}
+
+const UserInputForm = styled.div`
+  margin: 0px 24px;
+`;
+
+export const EtherUUID = '356a192b-7913-504c-9457-4d18c28d46e6';
+
+const ZapForm = (props: Props) => {
+  const { accounts } = useContext(StoreContext);
+  const { assets } = useContext(AssetContext);
+  const { networks } = useContext(NetworkContext);
+  const ethAsset = assets.find(asset => asset.uuid === EtherUUID) as ExtendedAsset;
+  const network = networks.find(n => n.baseAsset === EtherUUID) as Network;
+
+  const initialFormikValues: ISimpleTxFormFull = {
+    account: {} as IAccount,
+    amount: '',
+    asset: ethAsset,
+    nonce: '0',
+    gasPrice: '20',
+    address: '',
+    gasLimit: '',
+    network
+  };
+
+  const relevantAccounts = accounts.filter(isEthereumAccount);
+
+  return (
+    <>
+      <UserInputForm>
+        <Formik
+          initialValues={initialFormikValues}
+          onSubmit={fields => {
+            fetchGasPriceEstimates(fields.network).then(({ fast }) => {
+              console.debug('[here?]: ', fast);
```

Comment:
Should not be an issue for now but needs to be removed at some point since it will not show up next time we submit a PR.